### PR TITLE
Add thirty-six-fragments-of-midnight

### PR DESCRIPTION
**Pull request by HorrorTroll**

### CODE REVIEW

**Comment by AcK77** [comment]
The boxart.png it’s the game icon, not really a boxart. And NvMap means nothing. Can you change the boxart and add a more explicit error message. Thanks.
Next time you do a PR, if you can send a pastebin (or equivalent) with Ryujinx log inside, it would be great!

**Comment by HorrorTroll** [comment]
hmmm the games doesn't have boxart, i only find that game icon on internet!

Log: https://pastebin.com/kB4KfTUx

**Comment by AcK77** [comment]
boxart

Can you these one as boxart please (Don't judge my photoshop skills please :P).
Thanks for the log too, at least you should add `probably due to a rendering error` if the game don't print nothing after that.

**Comment by HorrorTroll** [comment]
Done

**Comment by AcK77** [comment]
The boxart isn’t in the right folder and you haven’t change the error message by a too explicit one...

File: 36-fragments-of-midnight/info.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	"name": "36 Fragments of Midnight",
 	"state": "Emulation Error",
 	"state_emu_version": "19683868",
-	"state_last_date": "11/07/2018",
-	"comment": "Nothing happens after NvMap 12 gets created",
+	"state_last_date": "12/07/2018",
+	"comment": "It's trying to call IAudioOut: AppendAudioOutBuffer and IAudioOut: GetReleasedAudioOutBuffer after NvMap 12 creates",
```
Comment (firewolfgames) [inline 36-fragments-of-midnight/info.json]: **grammar fix**
`after NvMap 12 gets created`

**Comment by AcK77** [comment]
The folder name need to be `thirty-six-fragments-of-midnight`

**Pull request merged**